cloudflare_rules/cloudflare_httpreq_bot_high_volume.py

```diff
@@ -0,0 +1,6 @@
+def rule(event):
+    # Bot scores are [1, 99] where scores < 30 indicating likely automated
+    # https://developers.cloudflare.com/bots/concepts/bot-score/
+    if event.get("BotScore", 100) >= 30:
```

Comment:
What do you think about excluding blocked events from this rule?

Reply:
That makes sense, yeah I'll make that change

Reply:
That's probably make this more useful

Reply:
As far as I can tell we can't tell if the event was blocked in the HTTP Req logs